doc/faq.rst

```diff
@@ -30,7 +30,7 @@ How do I...
    ``sidebartoc`` block.
 
 ... write my own extension?
-   See the :ref:`extension tutorial <exttut>`.
+   See the :ref:`extension tutorial <exttuto-todo>`.
```

Comment:
Instead of referencing this, how about:

    :doc:`/development/tutorial/index`

?